.env.example

```
@@ -0,0 +1,6 @@
+# Settings are stored in ~/.discord-search/settings.json
+# Priority: CLI args > environment variables > settings.json
+
+DISCORD_BOT_TOKEN=your_bot_token_here
+DISCORD_GUILD_ID=your_server_id_here
+DISCORD_CLIENT_ID=your_client_id_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISCORD_CLIENT_ID key should go before the DISCORD_GUILD_ID key

(UnorderedKey)